scripts/neighbor_advertiser

```diff
@@ -192,6 +193,39 @@ def get_vlan_addr(vlan_intf_name, ip_ver):
     return vlan_addr
 
 
+def get_vlan_addresses(vlan_interface):
+    vlan_id = get_vlan_interface_vlan_id(vlan_interface)
+    vxlan_id = get_vlan_interface_vxlan_id(vlan_interface)
+
+    mac_addr = None
+    ipv4_addr = []
+    ipv6_addr = []
+
+    try:
+        out = subprocess.check_output(["ifconfig", vlan_interface])
```

Comment:
Suggest to use `ip addr show` instead of `ifconfig `to get all the configured addresses

Reply:
Thanks! Addressed.